### PR TITLE
Move baldric from Medieval Stuff mod

### PR DESCRIPTION
**Pull request by Rail-Runner**
Move baldric from Medieval Stuff mod into the main game.

Other changes:
1. Simplify baldric recipe by combining two leather component entries into one.
2. Remove VARSIZE from sheaths and scabbards and divide their encumbrance by 2, rounding down.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -6671,6 +6671,33 @@
...
+        "price" : 7500,
+        "material" : ["leather", "wood"],
+        "volume" : 8,
+        "bashing" : 4,
+        "warmth" : 0,
+        "phase" : "solid",
+        "encumbrance" : 7,
+        "coverage" : 10,
+        "material_thickness" : 1,
+        "flags" : ["VARSIZE", "WAIST", "OVERSIZE"],
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Would be better without that VARSIZE and with 3 encumbrance IMO. I don't think scabbards need to be tailored to the wearer as much as boots or shirts.

Comment (Darkyhard) [inline data/json/items/armor.json]: Is this true for all simillar items, like other types of scabbards, sheaths and holsters?

Comment (Coolthulhu) [inline data/json/items/armor.json]: Holsters no longer have VARSIZE.

Scabbards currently do have it, but most likely should lose it too. Stuff that has adjustable belts and straps should not require tailoring to fit.

Comment (Darkyhard) [inline data/json/items/armor.json]: Done.

**Pull request merged**